R/get_parsing_tables.R

```diff
@@ -98,13 +98,17 @@ getParsingTables = function(tableSuffix) {
 
 
 createParsingTables = function(
-  tableSuffix = NULL, overwrite = FALSE, dbtype = 'postgres', dbname = NULL,
+  tableSuffix = NULL, overwrite = FALSE, dbtype = 'postgres', dbname = NULL, tabNames = c(),
   ...) {
   if (is.null(dbname)) return(invisible())
 
   con = connect(dbtype, dbname, ...)
   parTables = getParsingTables(tableSuffix)
 
+  if(!is.na(tabNames) && length(tabNames > 0)){
+    parTables = parTables[tabNames == names(parTables)]
```

Comment:
Use `names(parTables) %in% tableNames` inside the brackets. The current code will not work if length(tabNames) > 1.